runtime/compiler/env/j9method.h

```diff
@@ -457,6 +457,8 @@ class TR_ResolvedJ9Method : public TR_J9Method, public TR_ResolvedJ9MethodBase
    virtual TR_ResolvedMethod *     getResolvedPossiblyPrivateVirtualMethod( TR::Compilation *, int32_t cpIndex, bool ignoreRtResolve, bool * unresolvedInCP);
    virtual TR_OpaqueClassBlock *   getResolvedInterfaceMethod(int32_t cpIndex, uintptr_t * pITableIndex);
 
+   virtual TR_OpaqueMethodBlock *  getTargetMethodFromMemberName(uintptr_t * invokeCacheArray, uintptr_t & appendixObject, bool * isInvokeCacheAppendixNull = 0);
```

Comment:
In JITServer, and in in-process compilations when the caller doesn't hold VM access, the resulting `appendixObject` can't be used for any purpose other than to compare it against zero/null. AFAICT all call sites are correct, but IMO in the future somebody could easily think that `appendixObject` is actually meaningful as a reference. And it's redundant with `isInvokeCacheAppendixNull` anyway. What do you think of removing the former and making the latter mandatory?

Reply:
Yeah I think that makes sense, I'll make that change.